tests/console/perl_bootloader.pm

```diff
@@ -15,19 +15,22 @@ use warnings;
 use utils;
 use package_utils;
 use power_action_utils 'power_action';
-use version_utils qw(is_sle check_version is_transactional);
+use version_utils qw(is_sle is_leap is_sle_micro check_version is_transactional);
 use transactional;
 
 sub run {
     my ($self) = @_;
+    # https://progress.opensuse.org/issues/165686
+    # package name is now update-bootloader on Tumbleweed, it will remain perl-Bootloader on older products for the time being
```

Comment:
I suggest to also rephrase this text accordingly and not mention Tumbleweed but the negated inverse